ouroboros-consensus-test/test-storage/Test/Ouroboros/Storage/ChainDB/StateMachine/RunOnRepl.hs

```diff
@@ -0,0 +1,125 @@
+{-# LANGUAGE PatternSynonyms       #-}
+{-# LANGUAGE ConstraintKinds       #-}
+{-# LANGUAGE DataKinds             #-}
+{-# LANGUAGE DerivingVia           #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE FlexibleContexts      #-}
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE ScopedTypeVariables   #-}
+{-# LANGUAGE TypeFamilies          #-}
+{-# LANGUAGE UndecidableInstances  #-}
+
+-- | Collection functions and exported symbols to be able to run a
+-- 'quickcheck-state-machine' counterexample using the GHC repl.
+--
+-- To test a counterexample first fire up the repl, and enable the
+-- 'FlexibleContexts' and 'TypeFamilies' extensions, and set multi-line input.
+--
+-- > cabal new-repl test-storage
+-- > import Test.Ouroboros.Storage.ChainDB.StateMachine.RunOnRepl
+-- > :set -XFlexibleContexts -XTypeFamilies +m
+--
+-- The commands that are part of the counterexample are usually several lines
+-- long. Thus it is better to create a local definition for them:
+--
+-- > cmds = (<PASTE THE COMMANDS OF THE COUNTEREXAMPLE HERE>)
+--
+-- Note the use of parentheses to prevent GHCi from ending the multiline input
+-- prematurely (the copied counterexample in this case).
+--
+-- Then, the model and system under tests can be tested for lockstep agreement
+-- by running:
+--
+-- > quickCheckeCmdsLockStep someClockSkew someChunkInfo counterexample
+--
+-- Where 'someClockSkew' and 'someChunkInfo' are the ones given by a the
+-- counterexample found by quickcheck-statemachine.
+module Test.Ouroboros.Storage.ChainDB.StateMachine.RunOnRepl (
+    -- * Running the counterexamples
+    quickCheckeCmdsLockStep
+    -- * Patterns needed to disambiguate the 'At' and 'Command' symbols printed
+    -- * by the 'ChainDB.StateMachine' tests.
```

Comment:
typo: Remove leading `*` as I don't think you want another list item here

Reply:
Ups, `M-q` :grin: Thanks.